src/operators/fully-connected-nc.c

```diff
@@ -2020,6 +2189,23 @@ enum xnn_status xnn_reshape_fully_connected_nc_qp8_f32_qc4w(
       sizeof(fully_connected_op->params.f32_minmax), threadpool);
 }
 
+enum xnn_status xnn_reshape_fully_connected_nc_qp8_f32_qb4w(
+    xnn_operator_t fully_connected_op, size_t batch_size,
+    pthreadpool_t threadpool) {
+  return reshape_fully_connected_nc(
+      fully_connected_op, xnn_operator_type_fully_connected_nc_qp8_f32_qb4w,
+      batch_size,
+      /*log2_input_element_size=*/0,
+      // Pass 1 byte even though it is half byte, we handle the division via
+      // filter_is_nibble == true.
```

Comment:
remove this since we have it below?

Reply:
I think this line should be read with the comment line above it, which makes it quite useful.